### ISSUE TITLE
Is it possible to give an example of directly outputting the classification results?

### ISSUE THREAD

**Issue by snow-wind-001** [OML-Team/open-metric-learning#558]
https://open-metric-learning.readthedocs.io/en/latest/feature_extraction/python_examples.html
We used the example of the above URL, but there is no example of directly outputting two photos and outputting the classification results. In order to allow more people to use this project conveniently, please provide simple implementation code.

**Comment by snow-wind-001** [comment]
@chang48 @dapladoc @churnikov @alexmelekhin

**Comment by AlekseySh** [comment]
Hey, @snow-wind-001 
You can have classification labels only if you trained your model with classification loss (like arcface or crossentropy).
Note, all the models in Zoo have been trained using contrastive losses. There is no classification output by design in this case.

**Comment by snow-wind-001** [comment]
Can I understand that to put new photos into a list of one photo per category, I calculate the distances via pairwise_dist, or use pairwiseImgesPostprocessor to process the queue, when the queue torch.topk is at the front I find the same class.
@AlekseySh

**Comment by AlekseySh** [comment]
@snow-wind-001 in other words, you want to use kNN for classification, take a look on: https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsClassifier.html

Yep, it's possible. Note, the results significantly dependent on the probe images you picked. Also, it may be a good idea to have more than just one image per class.
